tests/manual-test-cases/Group5-Functional-Tests/5-4-High-Availability.md

```diff
@@ -17,8 +17,16 @@ This test requires access to VMWare Nimbus cluster for dynamic ESXi and vCenter
 ```govc cluster.change -drs-enabled -ha-enabled /ha-datacenter/host/cls```
 3. Deploy a new VCH Appliance to the cluster  
 4. Run a variety of docker commands on the VCH appliance
-5. Power off the ESXi host that the VCH is currently running on
-6. Run a variety of docker commands on the VCH appliance
+5. Create a named volume
+6. Create a container with a mounted anonymous and named volume
+7. Verify that the volumes are still there using inspect before powering off the ESXi
+
+8. Power off the ESXi host that the VCH is currently running on
+
+9. Verify that the volumes are still there using inspect after powering off the ESXi
+10. Clean up the created container (docker rm)
+
+11. Run a variety of docker commands on the VCH appliance
```

Comment:
@mhagen-vmware they were there before I added a couple of more lines to this area.  I think they may have been separating some "group" of actions.  I'll remove the spaces.